packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -479,7 +480,7 @@ class FixedExtentScrollPhysics extends ScrollPhysics {
   }
 
   @override
-  Simulation? createBallisticSimulation(ScrollMetrics position, double velocity) {
+  ScrollSimulation? createBallisticSimulation(ScrollMetrics position, double velocity) {
```

Comment:
Changes like this would also might be breaking and unable to be merged without a non-breaking migration strategy. The current test failures should be able to confirm this.